apps/emqx_retainer/src/emqx_retainer_schema.erl

```diff
@@ -27,9 +27,9 @@ fields(mnesia_config) ->
     ];
 
 fields(flow_control) ->
-    [ {max_read_number, sc(integer(), 0, fun is_pos_integer/1)}
-    , {msg_deliver_quota, sc(integer(), 0, fun is_pos_integer/1)}
-    , {quota_release_interval, sc(emqx_schema:duration_ms(), "0ms")}
+    [ {batch_read_number, sc(integer(), 0, fun is_pos_integer/1)}
+    , {batch_deliver_number, sc(range(0, 50), 0)}
```

Comment:
Can the upper limit be bigger? i.e: 1000?

Reply:
If the value is too large, it may cause difficulty in obtaining the token

Reply:
For a configuration item, we should try to guarantee our delivery capability as much as possible we can support. Although it will be discarded by the rate-limiting system. However, we should allow a large value in the configuration. This is more flexible.